index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />    <link rel="stylesheet" href="style.css">
+    <title>Cars-connect-254 conference website</title>
+</head>
+<body>
+    <div class=" wrapper">
+            <nav class="topper">
+                <ul class="top-nav">
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><i class="fa-brands fa-twitter"></i></li>
+                    <li><a href="#"></a>English</li>
+                    <li><a href="#"></a>MyPage</li>
+                    <li><a href="#"></a>Logout</li>
+                </ul>
+            </nav>
```

Comment:
- [ ] Good job adding this section above the navbar 👍 , however I noticed that you did not respect the margin and the padding for the section,  kindly make sure the margin, the padding, the spacing are respected because you should stick to the design , that means , make sure the margin, the padding , the color of your design should look as much as possible as in the template design

![Capture 1](https://user-images.githubusercontent.com/80458199/226099552-6cdb36fe-42a1-46b7-bd66-dbc6129bec5b.PNG)